optimize/qa/schema-integrity-tests/pom.xml

```diff
@@ -19,7 +19,7 @@
     <dependency>
       <groupId>org.elasticsearch.client</groupId>
       <artifactId>elasticsearch-rest-high-level-client</artifactId>
```

Comment:
@KarpovIvan - this module is newly committed since you started working on this project and therefore uses the old rest client. How would you like to proceed here? Note that it is only used for testing internally and is not product facing, but it would be nice to remove the module entirely if we can

Reply:
IMO During this merge I propose dont touch this dependency and fix it in next part of work.
Because I dont know how impact we will have after remove

Reply:
Let's make a note to create an issue to migrate this module after this PR. I expect it to be only a small amount of work